bcdaworker/queueing/enqueue.go

```diff
@@ -20,7 +21,13 @@ type Enqueuer interface {
 }
 
 func NewEnqueuer() Enqueuer {
-	return queEnqueuer{que.NewClient(database.QueueConnection)}
+	if conf.GetEnv("QUEUE_LIBRARY") == "que-go" {
+		return queEnqueuer{que.NewClient(database.QueueConnection)}
+	} else if conf.GetEnv("QUEUE_LIBRARY") == "river" {
+		return riverEnqueuer{}
+	} else {
+		panic("NO QUEUE LIBRARY SET")
+	}
```

Comment:
Not sure if panic-ing is better than just defaulting to a specific library?

Reply:
If we're going to a green-blue solution for the libraries I'd just default to que-go.

Reply:
yeah, I think defaulting to que-go would probably be best here in the event of a failure, since that's the existing behavior